gen3-client/g3cmd/upload-multiple.go

```diff
@@ -77,22 +79,45 @@ func init() {
 			}
 
 			furObjects := validateObject(objects, uploadPath)
-
 			if batch {
 				workers, respCh, errCh, batchFURObjects = initBatchUploadChannels(numParallel, len(objects))
 			}
 
 			for i, furObject := range furObjects {
-				if batch {
+				if forceMultipart {
+					uploadPath := []string{furObjects[i].FilePath}
+					_, multipartFilePaths := validateFilePath(uploadPath, forceMultipart)
+					log.Println(multipartFilePaths)
+					if len(multipartFilePaths) > 0 {
+						log.Println("Multipart uploading....")
+						for _, filePath := range multipartFilePaths {
+							fileInfo, err := ProcessFilename(furObjects[i].FilePath, filePath, false, false)
+							if err != nil {
+								logs.AddToFailedLog(filePath, filepath.Base(filePath), commonUtils.FileMetadata{}, "", 0, false, true)
+								log.Println("Process filename error for file: " + err.Error())
+								continue
+							}
+							err = multipartUpload(gen3Interface, fileInfo, 0, bucketName)
+							if err != nil {
+								log.Println(err.Error())
+							} else {
+								logs.IncrementScore(0)
+							}
+						}
+						if !logs.IsFailedLogMapEmpty() {
+							retryUpload(logs.GetFailedLogMap())
+						}
+					}
+				} else if batch {
 					if len(batchFURObjects) < workers {
 						batchFURObjects = append(batchFURObjects, furObject)
 					} else {
-						batchUpload(gen3Interface, batchFURObjects, workers, respCh, errCh)
+						batchUpload(gen3Interface, batchFURObjects, workers, respCh, errCh, bucketName)
```

Comment:
Also, I don't believe `multipartUpload` gets called at all if part of a batch? @BinamB